### DISCUSSION TITLE
Toolkit CornerRadiusEffect

### DISCUSSION THREAD

**Discussion by yurysemv** [CommunityToolkit/Maui#1795] [New Feature Discussions]
Hello! I'm updating applications from Forms to MAUI. It very often uses the xct:CornerRadiusEffect.CornerRadius effect. This effect is not yet available in the Toolkit. What's the best way to proceed? For now, I’m replacing everything with Boxview and Frame in Forms, then in MAUI I’ll switch to Border.

**Comment by CliffAgius** [comment]
Hi the reason it's not part of the MCT is that unlike Xamarin CornerRadius is now part of the default MAUI controls you can check the docs for how to achieve this [Frames](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/frame?view=net-maui-8.0) and [BoxView](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/boxview?view=net-maui-8.0)
